### PR TITLE
fix gc

### PR DESCRIPTION
**Pull request by JackTn**

### CODE REVIEW

**Comment by raych1** [comment]
Does version need to bump and release?

File: test/package-lock.json [hunk 1]
```diff
@@ -5545,7 +5569,14 @@
       "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
       "integrity": "sha1-ajhfuIU9lS1f8F0Oiq+UJ43GPc8=",
       "requires": {
-        "ansi-regex": "^5.0.1"
+        "ansi-regex": "^2.0.0"
```
Comment (raych1) [inline test/package-lock.json:5577]: Why do we downgrade this version?

**Comment by JackTn** [comment]
bump version

1. convict: 5.0.0 => 6.2.2

**Comment by JackTn** [comment]
bump to version 0.1.2

**Pull request merged**